package.json

```diff
@@ -5,7 +5,9 @@
   "main": "index.js",
   "scripts": {
     "watch": "webpack --watch -d & http-server dist/ & wait",
-    "build": "VERSION=`cat package.json | json version` ./scripts/build.sh",
+    "build-front": "VERSION=`cat package.json | json version` ./scripts/build.sh",
+    "build-back": "VERSION=`cat package.json | json version` ./scripts/build-admin.sh",
+    "build": "npm run build-front; npm run build-back;",
```

Comment:
Can you use command && command so that if one fail it stop (dunno if yours will stop also?)